src/app/components/toolbar/toolbar-menu-item/toolbar-menu-item.component.html

```diff
@@ -29,7 +29,7 @@
   <ng-container *ngSwitchDefault>
     <button
       [id]="actionRef.id"
-      mat-menu-item
+      [class]="'mat-menu-item'"
```

Comment:
are w sure this doesn't impact other things since `mat-menu-item` is a component with some internal  logic such as focus monitor? `mat-menu-item` can take also a role attribute 
```
...
<button
    mat-menu-item
    role="button" // this will override default menuitem role
</div>
...
```

Reply:
this works, thanks. it's not ideal markup-wise, but as you say it has the fewest potential impacts later. the additional role and the tabindex here are not needed, but also don't hurt anything.